dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/projection/Microsoft.ApplicationModel.Resources.Projection.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.2.1" />
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.2.3" />
```

Comment:
does this require sdk version change?

Reply:
it does not require an  SDK later than the current requirement of framework *.16 (which will become .NET 5.0.6) - no change to winrt.runtime.dll version, 1.2.0.0